D2211R0.md

```diff
@@ -82,19 +82,19 @@ below is one such pattern, but the literal `2` in the second arm is not.
 
 ```C++
 inspect(i) {
```

Comment:
Do we want to put a `-> std::ostream&` in these examples? To make them "compile" according to R3 paper.